rust/protocol/src/storage/inmem.rs

```diff
@@ -1,36 +1,46 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2022 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
+//! Implementations for stores defined in [super::traits].
+//!
+//! These implementations are purely in-memory, and therefore most likely useful for testing
+//! right now.
```

Comment:
Nitpick: let's strike the "right now". I don't think we generally want to give the impression that a bunch of features are "just around the corner" or anything.